compiler/src/iree/compiler/GlobalOptimization/PropagateLinalgTranspose.cpp

```diff
@@ -632,23 +632,58 @@ class FuseTransposeWithLinalgOpConsumer
   bool allowGeneralizing = false;
 };
 
-bool isUnaryElementwiseGeneric(linalg::GenericOp genericOp) {
-  if (genericOp.getNumDpsInputs() != 1 || genericOp.getNumDpsInputs() != 1 ||
-      !linalg::isElementwise(genericOp)) {
-    return false;
+static bool isOperandMapAffectedByTransposeMap(
```

Comment:
There is no `operand` semantic in the method. How about renaming it to `isIndexingMapAffectedByTransposeMap`?